File: python/Programmers_level/level1/최대공약수와 최소공배수.py
# Programmers 최대공약수와 최소공배수
def solution(n, m):
    answer = []
    a, b = n, m
    # 최대공약수 구하기
    while a % b != 0:
        a, b = b, a % b
    answer.append(b)
    # 최소공배수 구하기, n * m = L * G
    answer.append(n * m // b)
    return answer

File: Javascript/Programmers_level/level1/최대공약수와 최소공배수.js
function solution(n, m) {
  var answer = []
  let a = n,
    b = m
  // 최대공약수
  while (a % b !== 0) {
    let temp = a
    a = b
    b = temp % b
  }
  answer.push(b)

  // 최소공배수
  answer.push(parseInt((m * n) / b))
  return answer
}
